src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheck.java

```diff
@@ -82,8 +82,7 @@ public void visitJavadocToken(DetailNode ast) {
             if (textNode != null && textNode.getType() == JavadocTokenTypes.TEXT
                     && textNode.getChildren().length > 1) {
                 final DetailNode whitespace = JavadocUtils.getFirstChild(textNode);
-                if (whitespace.getType() == JavadocTokenTypes.WS
```

Comment:
at this point `vwhitespace.getType() == JavadocTokenTypes.WS` always returns `true`